dhis-2/dhis-web-api/src/main/java/org/hisp/dhis/webapi/controller/tracker/imports/MetadataIdentifierMapper.java

```diff
@@ -37,30 +46,46 @@
 {
 
     @Named( "programToMetadataIdentifier" )
-    default org.hisp.dhis.tracker.domain.MetadataIdentifier fromProgram( String identifier,
+    default MetadataIdentifier fromProgram( String identifier,
         @Context TrackerIdSchemeParams idSchemeParams )
     {
         return idSchemeParams.getProgramIdScheme().toMetadataIdentifier( identifier );
     }
 
     @Named( "programStageToMetadataIdentifier" )
-    default org.hisp.dhis.tracker.domain.MetadataIdentifier fromProgramStage( String identifier,
+    default MetadataIdentifier fromProgramStage( String identifier,
         @Context TrackerIdSchemeParams idSchemeParams )
     {
         return idSchemeParams.getProgramStageIdScheme().toMetadataIdentifier( identifier );
     }
 
     @Named( "orgUnitToMetadataIdentifier" )
-    default org.hisp.dhis.tracker.domain.MetadataIdentifier fromOrgUnit( String identifier,
+    default MetadataIdentifier fromOrgUnit( String identifier,
         @Context TrackerIdSchemeParams idSchemeParams )
     {
         return idSchemeParams.getOrgUnitIdScheme().toMetadataIdentifier( identifier );
     }
 
     @Named( "attributeOptionComboToMetadataIdentifier" )
-    default org.hisp.dhis.tracker.domain.MetadataIdentifier fromAttributeOptionCombo( String identifier,
+    default MetadataIdentifier fromAttributeOptionCombo( String identifier,
         @Context TrackerIdSchemeParams idSchemeParams )
     {
         return idSchemeParams.getCategoryOptionComboIdScheme().toMetadataIdentifier( identifier );
     }
+
+    @Named( "attributeCategoryOptionsToMetadataIdentifier" )
+    default Set<MetadataIdentifier> fromAttributeCategoryOptions( String identifiers,
+        @Context TrackerIdSchemeParams idSchemeParams )
+    {
+        if ( identifiers == null || StringUtils.isBlank( identifiers ) )
```

Comment:
```suggestion
        if ( StringUtils.isBlank( identifiers ) )
```

Comparison to null can be removed